resources/metadata.json

```diff
@@ -4,5 +4,5 @@
   "uuid": "peek-top-bar-on-fullscreen@marcinjahn.com",
   "shell-version": ["43", "44"],
   "url": "https://github.com/marcinjahn/gnome-peek-top-bar-on-fullscreen-extension",
-  "version": 7
+  "version": 8
```

Comment:
Please don't update the version yet. I usually do that in a separate commit.

Reply:
done